web/src/pages/RiyadesaiPage/RiyadesaiPage.js

```diff
@@ -0,0 +1,28 @@
+import { Link, routes } from '@redwoodjs/router'
+import { MetaTags } from '@redwoodjs/web'
+
+const RiyadesaiPage = () => {
+  return (
+    <>
+      <MetaTags title="Riyadesai" description="Riyadesai page" />
+
+      <body style="background-color:skyblue;">
```

Comment:
I don't believe you can use a <body> tag inside a React component.

Reply:
Yes, the tag doesn't work. I corrected it. Thank you.